src/probe.cpp

```diff
@@ -216,43 +231,46 @@ ProbeResult Probe::checkQuantumBreak() {
 		hasQBreakLevels && hasMicrosoftProducts &&
 		hasMonarchHQ && hasNissanLeaf && hasTimeMachine) {
 		spdlog::debug("Looks like Quantum Break!");
 		return kResultQuantumBreak;
 	} else {
 		spdlog::debug("Doesn't look like Quantum Break...");
+		ResMan.setPathPrefix(""); // reset pathPrefix
 		return kResultUnknown;
 	}
 }
 
 ProbeResult Probe::checkControl() {
+	ResMan.setPathPrefix(_pathPrefixes[kResultControl]);
 	// Check for Control missions
 	bool hasControlMissions =
-		ResMan.hasDirectory("data/worlds/gameworld/layers/hub_mission_01_the_bureau") &&
-		ResMan.hasDirectory("data/worlds/gameworld/layers/hub_mission_02_the_hotline") &&
-		ResMan.hasDirectory("data/worlds/gameworld/layers/maintenance_main_mission_03_lockdown") &&
-		ResMan.hasDirectory("data/worlds/gameworld/layers/research_mission_04_marshall") &&
-		ResMan.hasDirectory("data/worlds/gameworld/layers/maintenance_main_mission_05_blackrock") &&
-		ResMan.hasDirectory("data/worlds/gameworld/layers/containment_main_mission_06_dylan") &&
-		ResMan.hasDirectory("data/worlds/gameworld/layers/containment_main_mission_07_ordinary") &&
-		ResMan.hasDirectory("data/worlds/gameworld/layers/maintenance_main_mission_08_ahti") &&
-		ResMan.hasDirectory("data/worlds/gameworld/layers/research_mission_09_the_hedron") &&
-		ResMan.hasDirectory("data/worlds/gameworld/layers/hub_mission_10_nightmare");
+		ResMan.hasDirectory("worlds/gameworld/layers/hub_mission_01_the_bureau") &&
+		ResMan.hasDirectory("worlds/gameworld/layers/hub_mission_02_the_hotline") &&
+		ResMan.hasDirectory("worlds/gameworld/layers/maintenance_main_mission_03_lockdown") &&
+		ResMan.hasDirectory("worlds/gameworld/layers/research_mission_04_marshall") &&
+		ResMan.hasDirectory("worlds/gameworld/layers/maintenance_main_mission_05_blackrock") &&
+		ResMan.hasDirectory("worlds/gameworld/layers/containment_main_mission_06_dylan") &&
+		ResMan.hasDirectory("worlds/gameworld/layers/containment_main_mission_07_ordinary") &&
+		ResMan.hasDirectory("worlds/gameworld/layers/maintenance_main_mission_08_ahti") &&
+		ResMan.hasDirectory("worlds/gameworld/layers/research_mission_09_the_hedron") &&
+		ResMan.hasDirectory("worlds/gameworld/layers/hub_mission_10_nightmare");
 	// Check for mopping Ahti
 	bool hasAhti =
-		ResMan.hasResource("data/animations/intermediate/p7/human/male/ahti/ahti_mopping_loop_02.binanimclip");
+		ResMan.hasResource("animations/intermediate/p7/human/male/ahti/ahti_mopping_loop_02.binanimclip");
 	// Check for vending machine
 	bool hasVendingMachine =
-		ResMan.hasResource("data/objects/props/vending_machines/vending_machines_vending_machine.binfbx");
+		ResMan.hasResource("objects/props/vending_machines/vending_machines_vending_machine.binfbx");
 
 	if (hasControlMissions && hasAhti && hasVendingMachine) {
 		spdlog::debug("Looks like Control!");
 		return kResultControl;
 	} else {
 		spdlog::debug("Doesn't look like Control...");
+		ResMan.setPathPrefix(""); // reset pathPrefix
```

Comment:
I don't think you need to "reset" the path prefix here because you already set the final prefix later

Reply:
Indeed not strictly needed, but I believe it's good practice to clean up at the end of a function when the game is not detected. But sure, I deleted the resets as they are not needed now.